integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -41,25 +36,37 @@ def set_random_seed(seed=0):
 
 
 def compile_tf_module(tf_module,
-                      exported_names=(),
                       target_backends=(),
+                      exported_names=(),
                       artifacts_dir=None,
                       keep_saved_model=False):
   """Compiles a TensorFlow tf.Module and optionally saves compilation artifacts.
 
+  If artifacts_dir is provided then the following artifacts will be saved:
+    saved_model:
```

Comment:
it isn't clear from the wording here if `saved_model` is the saved model itself (which is a directory structure) or whether the saved model on disk format (which is a directory structure) is nested inside of here.

Reply:
Reworded to refer to is as a "TF SavedModel directory", similar to the wording [here](https://www.tensorflow.org/guide/saved_model#the_savedmodel_format_on_disk).